### PR TITLE
Stop sending a Bitcoin alert to Dogecoin clients

### PR DESCRIPTION
**Pull request by rnicoll**

### CODE REVIEW

**Review by patricklodder — COMMENTED** [review]
Can we change the documentation in the code?

File: src/net_processing.cpp [hunk 1]
```diff
@@ -2471,10 +2471,10 @@ void PeerManager::ProcessMessage(CNode& pfrom, const std::string& msg_type, CDat
         // If the peer is old enough to have the old alert system, send it the final alert.
         // TODO: Decide if we need to reintroduce alerts to Dogecoin
         //       https://github.com/dogecoin/dogecoin/issues/2231
```
Comment (rnicoll) [inline src/net_processing.cpp:2473]:
```suggestion
        // Dogecoin: Do not send a Bitcoin alert to Dogecoin clients.
```

**Comment by rnicoll** [comment]
Tweaked, how's that?

**Comment by rnicoll** [comment]
Closes #2231

**Review by patricklodder — APPROVED** [review]
ACK.

**Pull request merged**